pkg/infra/usagestats/service/usage_stats.go

```diff
@@ -230,6 +230,11 @@ func (uss *UsageStats) GetUsageReport(ctx context.Context) (usagestats.Report, e
 
 	metrics["stats.uptime"] = int64(time.Since(uss.startTime).Seconds())
 
+	featureUsageStats := uss.features.GetUsageStats(ctx)
```

Comment:
Would it be possible to use https://github.com/grafana/grafana/blob/725dbf8d95cdee03bdee12f168f9c84b185c1d6d/pkg/infra/usagestats/service.go#L18 https://github.com/grafana/grafana/blob/725dbf8d95cdee03bdee12f168f9c84b185c1d6d/pkg/infra/usagestats/service.go#L24 instead?

Reply:
Ah, yeah, let's merge this and I'll see if my refactor makes that possible then 🙂